libs/blocks/section-metadata/section-metadata.js

```diff
@@ -34,6 +36,37 @@ function handleTopHeight(section) {
   const headerHeight = document.querySelector('header').offsetHeight;
   section.style.top = `${headerHeight}px`;
 }
+
+export function promoIntersectObserve(el, startPoint, options = {}) {
+  let abovePromoStart = false;
+  const io = new IntersectionObserver((entries, observer) => {
+    entries.forEach(async (entry) => {
+      if (el.classList.contains('close-sticky-section')) return observer.unobserve(entry.target);
+      const isPromoStart = startPoint ? entry.target === startPoint : null;
+      abovePromoStart = isPromoStart
+        ? (entry.isIntersecting || entry.boundingClientRect.y > 0)
+        : abovePromoStart;
+      if (entry.isIntersecting || (!entry.isIntersecting && isPromoStart && abovePromoStart)) el.classList.add('hide-sticky-section');
```

Comment:
Instead of using a ternary operation that sets a boolean or null, you can simply use a logical AND
The whole condition for isIntersecting is a bit hard to read, maybe this simplifies it a bit:
```suggestion
      const isPromoStart = startPoint && entry.target === startPoint;
      abovePromoStart = isPromoStart ? (entry.isIntersecting || entry.boundingClientRect.y > 0) : abovePromoStart;
      if (entry.isIntersecting || (isPromoStart && abovePromoStart)) el.classList.add('hide-sticky-section');
```

Reply:
@mokimo I have made the requested changes.